Tests/scripts/hook_validations/secrets.py

```diff
@@ -259,20 +273,31 @@ def calculate_shannon_entropy(data):
         return 0
     entropy = 0
     # each unicode code representation of all characters which are considered printable
-    for x in (ord(c) for c in string.printable):
+    for char in (ord(c) for c in string.printable):
         # probability of event X
-        px = float(data.count(chr(x))) / len(data)
-        if px > 0:
+        p_x = float(data.count(chr(char))) / len(data)
+        if p_x > 0:
             # the information in every possible news, in bits
-            entropy += - px * math.log(px, 2)
+            entropy += - p_x * math.log(p_x, 2)
     return entropy
 
 
-def get_white_list():
-    with io.open('./Tests/secrets_white_list.json', mode="r", encoding="utf-8") as secrets_white_list_file:
-        final_white_list = []
-        ioc_white_list = []
-        files_while_list = []
+def get_white_listed_items(is_pack, pack_name):
+    if is_pack:
+        packs_whitelist_path = PACKS_PATH + pack_name + PACKS_WHITELIST_RELATIVE_PATH
```

Comment:
use `os.path.join` and not hard-coded `/`

Reply:
Sorry you had to see this 😞